Write survey (#19)
KatieMurphy98/Year-4-Project: tex/chapters/survey/main.tex

\chapter{Data collection}

\section{Background on survey design}

> %cite: https://books.google.co.uk/books?id=mSRTDwAAQBAJ&lpg=PP1&ots=nynKGurXVG&dq=how%20to%20write%20a%20survey&lr&pg=PP1#v=onepage&q&f=false
> %cite: https://www.idsurvey.com/en/advantages-and-disadvantages-of-phone-survey/

Reviewer: ```suggestion
% TODO cite: https://books.google.co.uk/books?id=mSRTDwAAQBAJ&lpg=PP1&ots=nynKGurXVG&dq=how%20to%20write%20a%20survey&lr&pg=PP1#v=onepage&q&f=false
% TODO cite: https://www.idsurvey.com/en/advantages-and-disadvantages-of-phone-survey/
```

`# TODO` or `% TODO` or etc... Is a common convention for this sort of stuff (people often run commands to check that they can't find any files with any remaining `TODO`s left) :).

Also, FYI this is a GitHub "suggestion" you can directly commit the change on GitHub (you'll need to pull it down locally) which is a nice way of small little things getting fixed as I can show you exactly what I mean as opposed to describing it.

That doesn't mean you *have* to commit the suggestion: just that you can.

It's also possible to batch accept multiple suggestions at a time which helps keep the commits nice and clear :)

> A survey is a method of data collection with the purpose of obtaining quantitative information regarding opinions and 

Reviewer: We'll need a reference here. `... experiences of the respondents~\cite{...}.`

experiences of the respondents. Surveys could be used to investigate, for example, a product for which a company would like 
consumer feedback, or how a disease impacts patients in their daily lives. This form of investigation is useful for gathering a

> large amount of data that can be easily quantified and analysed using statistical methods and tests. 

Reviewer: Can you add a sentence or two with some examples of "famous" survey led results.

Survey methodology involves asking a sample of the target population a series of standardised questions, which may be given in 
the form of written questionaires or spoken interviews. Before the type of survey is decided, it is important to consider the 
aims and objectives of the research. 

For example, phone calls and other forms of interview-based survey allow the interviewer to form a personal connection with 
the survey participant, which can be especially helpful for a company's image if the interviewer is particularly professional 
or charismatic. This type of survey also provides an instant response, which is beneficial if there is only a short time frame 
available in which to gather information. Furthermore, any error as a result of participants misinterpreting questions can be 
negated due to the interviewer having the ability to immediately clarify on any misunderstandings. 

> However, there are also shortfalls to an interview-based survey method.

Reviewer: Let us discuss some of them here.

% About Questionaires
A questionaire is composed of two types of questions; closed-ended questions, where participants select answers from a list,
and open-ended questions, in which the participants are able to write their own answers. 

> \section{Specific goals of survey for this chapter}

Reviewer: ```suggestion
\section{Specific goals of survey tool for this dissertation}
```

\section{The survey}

\textbf{Cohorts}
\begin{itemize}

>     \item Students

Reviewer: We'll need to be more specific here.

    \item People who work in data visualisation and/or statistical programming

>     \item General population

Reviewer: We'll need to be more specific here.

\end{itemize}

\textbf{Information about participants}

\begin{itemize}

>     \item Please select your age category. (Checkbox: Under 18, 18-25, 26-35, 36-45, Above 45)

Reviewer: Any reason for these categories? Would just having the exact age be better?

>     \item If you are a university student or past university graduate please specify your area of study. (maybe drop down box: Science, Technology, Engineering, Mathematics, Arts, Social Sciences, Humanities, Business, Other, N/A)

Reviewer: Good idea.

>     \item How would you rate your spatial awareness skills? (Scale/slider: Poor, Okay, Good, Very Good)

Reviewer: I wonder if we'll need a short note about what "spatial awareness skills" are? Or is it better to leave it vague?

>     \item How would you rate your observation skills? (Scale/slider: Poor, Okay, Good, Very Good)

Reviewer: Same comment about awareness skills.

I wonder if both of these questions should be written in the style.

> How strongly do you agree with the following statement:

> "I consider myself to have above average spatial awareness"

But we'd need to know why it is that people use that form for questions.

>     \item How would you rate your mathematics skills? (Scale/slider: Poor, Okay, Good, Very Good)

Reviewer: Do we want to say mathematics or do we want to say numeracy? (Or both)

>     \item Do you consider yourself to be colourblind? (Binary checkbox: Y/N)

Reviewer: I am ignorant here and I want to make sure the questionnaire is as inclusive as possible.

Is colourblindness something one self declares? Or should we specify people for whom it is diagnosed? (If that even happens). 

Can you find some information and/or examples of how these questions are/should be asked.

>     \item Do you consider yourself to have a visual processing issue (ie. dyslexia)? (Binary checkbox: Y/N)

Reviewer: Again declaring ignorance here but same comment as the question on colour blindness. I'm not entirely sure "issue" is an inclusive term. **But** I'm 100% ignorant and just want to make sure we don't make a mistake: so let's find some examples/texts that show how these (important) questions are asked.

\end{itemize}

\noindent \textbf{Type of questions}

\begin{itemize}

>     \item Take categorical data and show different types of plot with no numbers and as questions such as "which would you say is the largest" etc - Pie chart, bar chart, pictogram.

Reviewer: Also: after a given section like this add a question about preference. (Declared preference might not correspond to evidenced effectiveness which would be neat.)

\section{Background on survey design}

%TODO 1 cite: https://books.google.co.uk/books?id=mSRTDwAAQBAJ&lpg=PP1&ots=nynKGurXVG&dq=how%20to%20write%20a%20survey&lr&pg=PP1#v=onepage&q&f=false
%TODO 2 cite: https://www.idsurvey.com/en/advantages-and-disadvantages-of-phone-survey/
%TODO 3 cite: https://books.google.co.uk/books?hl=en&lr=&id=ctow8zWdyFgC&oi=fnd&pg=PR15&dq=survey+methodology&ots=fgcIbBXkS9&sig=LvdOfWOBGrtHFrcVbdfi1ahV970#v=onepage&q=survey%20methodology&f=true
%TODO 4 cite: https://en.wikipedia.org/wiki/Census

As described in [CITE 3 HERE], surveys are used for obtaining quantitative information regarding opinions and 
experiences of the respondents in order to explore the views of the target population as a whole. A survey is a systematic

> method of collecting data, where the word "systematic" is deliberately used by the author of [CITE 3 HERE] in order to separate

Reviewer: ```suggestion
method of collecting data, where the word ``systematic" is deliberately used by the author of [CITE 3 HERE] in order to separate
```

In LaTeX you need to use `` for left quotation marks.

    (checkbox and open answer box: No, Yes (please specify))
\end{itemize}

%Data from: https://data.world/ninja/anw-obstacle-history

> The following data shows data relating to 5 of the most used obstacles from the TV show American Ninja Warrior. The questions

Reviewer: Cool :)
